rfcs/0000-add-host-fields.md

```diff
@@ -0,0 +1,128 @@
+# 0000: Name of RFC
```

Comment:
Please enter the title here as well.

But leave the ID at 0000, we'll attribute it at merge time :-)

Reply:
BTW I'd like us to be a bit more specific on RFC title, and mention this is about metrics. Something like:

> Host Metric Fields

& `rfcs/text/0000-host-metric-fields.md`